js/adapt-contrib-tutor.js

```diff
@@ -9,25 +9,28 @@ class Tutor extends Backbone.Controller {
   initialize() {
     this.listenTo(Adapt, {
       'componentView:postRender': this.onComponentViewPostRender,
-      'questionView:showFeedback': this.onQuestionViewShowFeedback
+      'questionView:showFeedback': this.onQuestionViewShowFeedback,
+      'buttonsView:postRender': this.onButtonsViewPostRender
     });
   }
 
   onComponentViewPostRender(view) {
-    const model = view.model;
+    const { model } = view;
+    const shouldShowFeedback = (model.get('_canShowFeedback') && model.get('_isSubmitted'));
+    if (!shouldShowFeedback) return;
+    model.setupFeedback();
+    Adapt.trigger('questionView:showFeedback', view);
+  }
+
+  onButtonsViewPostRender(view) {
+    const { model } = view;
     if (!model.isTypeGroup('question')) return;
-    const config = model.get('_tutor');
+    const config = !model.get('_tutor')?._isInherited
```

Comment:
```suggestion
    const config = model.get('_tutor')?._isInherited
```
The behaviour wasn't normalised when `_isInherited` on the component as it was pulling in the wrong config.

Reply:
Have sorted in a slightly better way. Apparently there isn't one truthy or falsey expression which fits, my bad.